cmd/boot-script-service/oauth.go

```diff
@@ -143,32 +203,38 @@ func QuoteArrayStrings(arr []string) []string {
 // Returns the OAuthClient struct containing the client ID, secret, etc. as well
 // as the access token and an error if one occurred.
 func (client *OAuthClient) RequestClientCreds() (accessToken string, err error) {
-	var (
-		url  string
-		resp []byte
-	)
+	return client.FetchAccessToken(oauth2AdminBaseURL + "/admin/clients")
```

Comment:
With this, it looks like `oauth2PublicBaseURL` is no longer used (defined [here](https://github.com/OpenCHAMI/bss/blob/e832626f80e1163e91eca97b75bfd2ea61cfcaf9/cmd/boot-script-service/main.go#L106C2-L106C21)). Will we still need it?